engine/src/main/java/org/hibernate/search/util/logging/impl/Log.java

```diff
@@ -1000,4 +1000,7 @@
 	@Message(id = 324, value = "The fieldBridge for field '%1$s' is an instance of '%2$s', which does not implement TwoWayFieldBridge. Projected fields must have a TwoWayFieldBridge.")
 	SearchException projectingFieldWithoutTwoWayFieldBridge(String fieldName, Class<?> fieldBridgeClass);
 
+	@Message(id = 325, value = "Index names collision detected. Different logical indexes will point to the same actual index: %1$s")
```

Comment:
I couldn't find any place where we use the terms "logical indexes". Ok with you if I replace that with "index managers"?